src/Microsoft.Diagnostics.Runtime/Microsoft.Diagnostics.Runtime.csproj

```diff
@@ -30,4 +30,10 @@
       <PackagePath></PackagePath>
     </Content>
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Buffers" Version="4.5.0" />
```

Comment:
Unfortunately ClrMD 1.1 has a strict requirement that it doesn't carry dependencies.  These libraries have to be removed.

Reply:
Thanks for pointing this out - I have addressed the feedback.